.ipynb_checkpoints/Deque.py

```diff
@@ -0,0 +1,80 @@
+import sys
+
+
+class Deque(object):
+    def __init__(self):
+        self.Size = 100
+        self.array = [0] * 100
+        self.Front = 0
+        self.end = 0
+        self.actualSize = 0
+
+    def push_front(self, n):
+        self.Front -= 1
+        self.array[self.Front] = n
+        self.actualSize += 1
+        print("ok")
+
+    def push_back(self, n):
+        if (self.actualSize != 0):
+            self.end += 1
+        self.array[self.end] = n
+        self.actualSize += 1
```

Comment:
if self.end == self.Size - 1 you will get an exception here

Reply:
Clear